sdk/src/loom2d/Display/Graphics.ls

```diff
@@ -1,20 +1,10 @@
-// =================================================================================================
-//
-//  Starling Framework
-//  Copyright 2011 Gamua OG. All Rights Reserved.
-//
-//  This program is free software. You can redistribute and/or modify it
-//  in accordance with the terms of the accompanying license agreement.
-//
-// =================================================================================================
-
 package loom2d.display
 {
     import loom2d.math.Rectangle;
     
-    /** TODO
+    /**
+     * Text alignment constants used by the Graphics class.
```

Comment:
Useful to explain that horizontal and vertical alignment can be used together with the | (OR) operator.